addons/sale/wizard/sale_make_invoice_advance.py

```diff
@@ -69,16 +69,7 @@ def onchange_advance_payment_method(self):
             return {'value': {'amount': 0}}
         return {}
 
-    def _create_invoice(self, order, so_line, amount):
-        if (self.advance_payment_method == 'percentage' and self.amount <= 0.00) or (self.advance_payment_method == 'fixed' and self.fixed_amount <= 0.00):
-            raise UserError(_('The value of the down payment amount must be positive.'))
-        if self.advance_payment_method == 'percentage':
-            amount = order.amount_untaxed * self.amount / 100
-            name = _("Down payment of %s%%") % (self.amount,)
```

Comment:
just an FYI, this `,` was not an accident (I see you removed it in the method you split up).

```py
>>> isinstance(('some variable'), tuple)
False
>>> isinstance(('some variable',), tuple)
True
```
In this case there's a single element for the translation and the translation method doesn't mind, some string extrapolation-like objects do mind though

Just something to keep in mind, it's probably better to leave such an extra comma where you find it in future cases ;à

Reply:
Thanks for the info, TIL, I didn't know that! 😉